### ISSUE TITLE
Fullscreen mode: show system bars, but hide answer buttons

### ISSUE THREAD

**Issue by eu2021** [ankidroid/Anki-Android#10319]
**Is your feature request related to a problem? Please describe.**
I use gestures or a controller, so I don't want to see the answer buttons. But, I need to see the system bars, because I use flags a lot, and the best way to assign flags and see which ones are assigned to a card is looking at the system bars.

**Describe the solution you'd like**
A fullscreen mode that shows system bars, but hide answer buttons.

Thanks,

**Comment by mikehardy** [comment]
Hi there! We already have quite a proliferation of different full screen states, I'm hesitant to add another.
You mention you use gestures a lot. Have you considered using the 9-point gestures system available (I think in the alphas?) such that you may map flag addition to gestures as well?
Have you considered using a template modification to add flag buttons to your card template via the javascript API? https://github.com/ankidroid/Anki-Android/wiki/AnkiDroid-Javascript-API#flag--remove-flag-in-current-card

**Comment by eu2021** [comment]
I went to look again at the Ankidroid options and found that option called "Show Top Bar". I also noticed that the fullscreen modes mention "**_system_** bars". So, my line of thought was: so, if you want to make the **_system_** bar hide, but keep the **_Ankidroid_** Top Bar showing, then use the fullscreen, and activate the option "Show Top Bar". Unfortunatelly, it doesn't work.

After that, I searched for "Show Top Bar" in the forum, and found that david-alisson already tried to solve this (#7058). So, I will close this request here, and show my interest there.

> You mention you use gestures a lot. Have you considered using the 9-point gestures system available (I think in the alphas?) such that you may map flag addition to gestures as well?

I like to split the screen in two huge buttons (half top and half bottom), as I use only 2 answer buttons (Again and Good). And I use all the 7 flags (and the Mark, suspend, bury and edit), so assigning gestures for everything wouldn't work for me.

> Have you considered using a template modification to add flag buttons to your card template via the javascript API?

This sounds interesting, but a bit overkill, as I would need to insert lots of buttons in lots of different card templates (I use many different premade decks).

Thanks,
